test/processors/file_processor_test.rb

```diff
@@ -1,7 +1,16 @@
 require 'test_helper'
 
 class FileProcessorTest < ActiveSupport::TestCase
-  let(:processor) { FileProcessor.new(logger: Logger.new('/dev/null')) }
+
+  let(:audio_monster) do
+    Minitest::Mock.new
+  end
+
+  let(:processor) do
+    FileProcessor.new(logger: Logger.new('/dev/null')).tap do |p|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.